sportsdataverse/nfl/nfl_player_stats.py

```
@@ -0,0 +1,111 @@
+
+import pandas as pd
+import polars as pl
+
+from sportsdataverse._common_espn_player_stats import _espn_player_stats
+
+_SPORT_SLUG: str = "football"
+_LEAGUE_SLUG: str = "nfl"
+
+
+@overload
+def espn_nfl_player_stats(
```

Flake8 (7.3.0)

[error] multiple statements on one line (def)

(E704)